src/utils/copyText.ts

```diff
@@ -23,19 +25,21 @@ function fallbackCopyTextToClipboard(text: string) {
   }
 }
 
-const copyToClipboard = async (text: string | null) => {
+const copyToClipboard = async (text: string | null): Promise<Result<null>> => {
```

Comment:
I think this (and other similar functions) should be `Result<void>`. This would indicate that there is no data ever produced by these functions. `Result<null>` is similar but it implies that there happens to be no data for a result (but there could be). This is useful for when you are getting a value but there may not be one e.g. `Result<string | null>`.